Dockerfile

```diff
@@ -20,10 +21,12 @@ ENV SERVICE_TAGS=webapp \
     SMEAGOL_HOME=/var/lib/smeagol
 
 COPY --from=builder /usr/src/smeagol/target/smeagol.war /app/smeagol.war
-COPY ces-startup.sh /app/startup.sh
+COPY resources/ /app/
```

Comment:
Is there a reason why the startup.sh is under `/app`?
It is usually located directly in `/`.